### PR TITLE
Update JS Release Notes for v6.0.5

### PR DESCRIPTION
**Pull request by dimitrisniras**

### CODE REVIEW

File: _documentation/client-sdk/sdk-documentation/javascript/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ navigation_weight: 0
 
 # Release Notes
 
+## Version 6.0.5 - November 19, 2019
+
+### Fixes
+
+- Update styling of jsdocs to docstrap template
+- Change ice gathering process to send sdp on first ice candidate gathered
```
Comment (tbedford) [inline _documentation/client-sdk/sdk-documentation/javascript/release-notes.md]: Whats is "ice"? Also what is "sdp"? Thanks.

Comment (tbedford) [reply _documentation/client-sdk/sdk-documentation/javascript/release-notes.md]: @dimitrisniras Should sdp be SDP? Also what does it stand for? Thanks!

Comment (dimitrisniras) [reply _documentation/client-sdk/sdk-documentation/javascript/release-notes.md]: Hey I just saw this PR. I will update the CHangelog with the fixes.

File: _documentation/en/client-sdk/sdk-documentation/javascript/release-notes.md [hunk 2]
```diff
@@ -6,12 +6,18 @@ navigation_weight: 0
...
+- Add `iceGatherOnlyOneCandidate` configuration option and use to define path in `ICE gathering` process
+
 ## Version 6.0.5 - November 19, 2019
 
 ### Fixes
 
-- Update styling of JavaScript documentation to DocStrap template
-- Change ice gathering process to send sdp on first ice candidate gathered
+- Update styling of `JSDocs` to `Docstrap` template
+- Change `RTCPeerConnection ICE candidates` gathering process to send the Sesssion Description Protocol (`SDP`) offer on first `ICE` candidate gathered
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/javascript/release-notes.md]: "Sesssion" Description Protocol -> Session ...

**Pull request merged**